tweepy/cursor.py

```diff
@@ -121,13 +128,18 @@ def next(self):
             else:
                 result = model
 
-            if len(self.results) != 0:
+
+            if self.results != 0:
                 self.index += 1
-            self.results.append(result)
-            self.model_results.append(model)
+            self.results += 1
+            self.model_results += 1
         else:
+            print("DON'T KNOW WHAT TO DO!!!!")
```

Comment:
really?

Reply:
Sorry, this was a personal branch I accidentally committed to this PR lol